### PR TITLE
Update reactor-netty-http timers to use the new Observation

### PR DESCRIPTION
**Pull request by violetagg**
HttpClient/HttpServer response times now are Observation

Related to #1952

Co-authored-by: Marcin Grzejszczak <mgrzejszczak@vmware.com>

### CODE REVIEW

**Comment by violetagg** [comment]
@reactor/netty-team PTAL

File: reactor-netty-http/src/test/java/reactor/netty/http/observability/ObservabilitySmokeTest.java [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+		byte[] bytes = new byte[1024 * 8];
+		Random rndm = new Random();
+		rndm.nextBytes(bytes);
+		content = new String(bytes, Charset.defaultCharset());
+
+		Metrics.addRegistry(simpleMeterRegistry);
+	}
+
+	@AfterEach
+	void setupRegistry() {
```
Comment (simonbasle) [inline reactor-netty-http/src/test/java/reactor/netty/http/observability/ObservabilitySmokeTest.java:75]: nit: technically more like `cleanRegistry()`

**Review by simonbasle — APPROVED** [review]
LGTM

File: reactor-netty-http/src/test/java/reactor/netty/http/observability/ObservabilitySmokeTest.java (hunk 1 above)
Comment (violetagg) [reply reactor-netty-http/src/test/java/reactor/netty/http/observability/ObservabilitySmokeTest.java]: fixed

**Review by pderop — APPROVED** [review]
LGTM (please check missing javadoc for new "enableTracing" parameter added to the SniProvider.addSniHandler method ?)

File: reactor-netty-core/src/main/java/reactor/netty/tcp/SniProvider.java [hunk 2]
```diff
@@ -41,15 +41,15 @@
 	 * @param channel the channel
 	 * @param sslDebug if true SSL debugging on the server side will be enabled
 	 */
-	void addSniHandler(Channel channel, boolean sslDebug) {
+	void addSniHandler(Channel channel, boolean sslDebug, boolean enableTracing) {
```
[the remark this answers is not in the document]
Comment (simonbasle) [reply reactor-netty-core/src/main/java/reactor/netty/tcp/SniProvider.java:44]: I would say yes, for our future selves :)

[the remark this answers is not in the document]
Comment (violetagg) [reply reactor-netty-core/src/main/java/reactor/netty/tcp/SniProvider.java]: fixed

**Comment by violetagg** [comment]
@pderop @simonbasle Thanks for the review!

**Pull request merged**